CreateSurfaceWindowsImage.ps1

```diff
@@ -146,79 +153,86 @@ Param(
         )]
         [bool]$AdobeFlashUpdate = $True,
 
-    [Parameter(
+        [Parameter(
         Position=10,
         Mandatory=$False,
+        HelpMessage="Add latest Out-Of-Band/Non Security update (bool true/false, default is true)"
+        )]
+        [bool]$OOBUpdate = $False,

```

Comment:
Minor: OOB feels too close to OOBE (out of box experience). Not sure how common the term is used in the industry though...

Reply:
supplimental update?